src/components/HomepageFeatures.module.css

```diff
@@ -0,0 +1,15 @@
+.features {
+  display: flex;
+  align-items: center;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+.featureSvg {
+  height: 200px;
+  width: 200px;
+}
```

Comment:
Not sure if we'd ever use the SVGs on the homepage. So we don't need them in the CSS.

```suggestion
```

Reply:
done!